src/windows/FileProxy.js

```diff
@@ -256,7 +256,7 @@ function pathFromURL(url) {
         }
     });
     
-    return path.replace('%3A',':').replace(driveRE,'$1');
```

Comment:
Decoding of the colon (and any other % escape codes) will be done by the decodeURIComponent() call I added above.